backend/windmill-api/src/concurrency_groups.rs

```diff
@@ -168,100 +138,142 @@ async fn get_concurrent_intervals(
     Extension(db): Extension<DB>,
     Extension(user_db): Extension<UserDB>,
     Path(w_id): Path<String>,
-    Query(iq): Query<ConcurrentIntervalsParams>,
+    Query(iq): Query<ExtendedJobsParams>,
     Query(lq): Query<ListCompletedQuery>,
-) -> JsonResult<ConcurrencyIntervals> {
+) -> JsonResult<ExtendedJobs> {
     check_scopes(&authed, || format!("listjobs"))?;
 
+    if lq.success.is_some() && lq.running.is_some_and(|x| x) {
+        return Err(error::Error::BadRequest(
+            "cannot specify both success and running".to_string(),
+        ));
+    }
+
+    const QJ_FIELDS: &[&str] = &[
```

Comment:
is that different from the ones of list_jobs ? Can we make it a common const if yes